site/docs/components/drawer/usage.mdx

```diff
@@ -0,0 +1,49 @@
+---
+# Leave the frontmatter as is
+title:
+  $ref: ./#/title
+layout: DetailComponent
+sidebar:
+  label: Usage
+data:
+  $ref: ./#/data
+---
+
+### Using the Component Name component
+
+Use this component by wrapping it around your content and setting the desired position. For example:
+
+```jsx
+<Drawer isOpen={open} position="right">
+  <DrawerContentExample />
+</Drawer>
+```
+
+Provide a component like a Button so that users can open and close the Drawer.
+
+The Drawer displays a visual overlay, also known as a scrim, over the existing content. It also plays relevant animations when opening and closing. You can disable all of these visuals via props.
```

Comment:
Is there a need to point out scrim is not only a "visual" thing but also focus management for accessibility?